example-mojaloop-backend/values.yaml

```diff
@@ -259,28 +149,9 @@ centralledger-obj:
   ##
   affinity: {}
 
-  ## @param nodeSelector Node labels for pod assignment
```

Comment:
Why removed?

Reply:
Per the earlier comment I am trying to simplify the values.yaml by consistently defaulting to the bitnami default.